src/web/command_control/client/components/RCControllerPanel.tsx

```diff
@@ -192,63 +196,73 @@ export default class RCControllerPanel extends React.Component {
 		})
 	}
 
+	/**
+	 * This handles the input from the xbox controller by mapping the value
+	 * of the inputs to a bin number and direction to give the user feedback
+	 * @param axisName string - indicates the analog stick that is being controlled
+	 * @param value number - indicates the position of the analog stick
+	 */
 	handleGamepadAxisChange(axisName: string, value: number) {
 		const controlType = this.state.controlType
 
-		let throttleDirection = ''
-		let rudderDirection = ''
-		let throttleBinNumber = 0
-		let rudderBinNumber = 0
+		let throttleDirection = this.state.throttleDirection
+		let rudderDirection = this.state.rudderDirection
+		let throttleBinNumber = this.state.throttleBinNumber
+		let rudderBinNumber = this.state.rudderBinNumber
+
+		let valuePercent = (value * 100)
+		let deadzone = 15
 
+		// Rudder Handler
 		if (axisName === (controlType === ControlTypes.MANUAL_SINGLE ? 'LeftStickX' : 'RightStickX')) {
 			let bin: {binNumber: number, binValue: number} = {binNumber: 0, binValue: 0}
-			this.calcRudderBinNum((value * 100), bin)
-			this.props.remoteControlValues.pid_control.rudder = bin.binValue
 			// Added a deadzone
-			if (value > 10) {
+			if (valuePercent > deadzone) {
 				rudderDirection = 'RIGHT'
-			} else if (value < -10) {
+				this.calcRudderBinNum(valuePercent, bin)
```

Comment:
Let's move these lines out of the nested conditional so we don't have to repeat them